packages/select/src/vaadin-lit-select-list-box.js

```diff
@@ -36,9 +37,15 @@ class SelectListBox extends ListMixin(ThemableMixin(DirMixin(PolylitMixin(LitEle
 
       [part='items'] {
         height: 100%;
-        width: 100%;
         overflow-y: auto;
-        -webkit-overflow-scrolling: touch;
+        width: 100%;
+      }
+
+      [part='items'] ::slotted(hr) {
+        border-color: var(--vaadin-divider-color, var(--_vaadin-border-color));
+        border-width: 0 0 1px 0;
```

Comment:
Let's use the shorthand to fix linter warning:

```
packages/select/src/vaadin-lit-select-list-box.js
  46:23  ✖  Expected "0 0 1px 0" to be "0 0 1px"  shorthand-property-no-redundant-values
```

```suggestion
        border-width: 0 0 1px;
```